vulnerable_people_form/form_pages/nhs_login_link.py

```diff
@@ -11,4 +11,5 @@ def get_nhs_login_link():
         "nhs-login-link.html",
         previous_path=dynamic_back_url(),
         nhs_login_href=current_app.nhs_oidc_client.get_authorization_url(),
+        continue_url="/postcode-eligibility"
```

Comment:
I am wondering if continue_url can have a better name to reflect what the url actually is.

Reply:
the continue_url is for the continue button